Sourcery/Generating/Generator.swift

```diff
@@ -25,7 +25,7 @@ private class TypesReflectionBox: NSObject {
     lazy var classes: [Type] = {
         return self.types.filter { type in
             let isNotClass = type is Struct || type is Enum || type is Protocol
-            return !isNotClass
+            return !isNotClass && !type.isExtension
```

Comment:
@ilyapuchka while you are at it, can you add 3 simple generator tests to prove that scenario for accessing this kind of data is working as expected, and so we can avoid regressions? I'll merge this and release a new version once that's done